staging/src/k8s.io/sample-apiserver/artifacts/example/rc.yaml

```diff
@@ -19,7 +19,6 @@ spec:
       - name: wardle-server
         image: kube-sample-apiserver:latest
         imagePullPolicy: Never
-        args:
```

Comment:
We didn't set "RUN" instruction in Dockerfile. So we need to use `command` here.